platform/projects_and_organizations/project_settings.mdx

```diff
@@ -8,15 +8,18 @@ import { ScreenshotWrapper } from '../shared/_ScreenshotWrapper';
 
 In project settings, you can edit the basic properties of your project.
 
-- **project name**
-- **description** - displayed in the project dashboard (can be formatted with Markdown)
-- **base language** - the language that is used as a base for translating to other languages
-- **default namespace** - preselected namespace when creating a new key
+- **Project Name**
+- **Description** - displayed in the project dashboard (can be formatted with Markdown)
+- **Base language** - the language that is used as a base for translating to other languages
+- **Default namespace** - preselected namespace when creating a new key
+
+To use namespaces in your project, you need to have namespaces enabled. Toggle "Use namespaces" to choose your default namespace. 
+You can read our article about [namespaces](https://docs.tolgee.io/js-sdk/namespaces) to learn more.

```

Comment:
I wouldn't link here to the js-sdk docs, instead, link here: https://docs.tolgee.io/platform/translation_keys/namespaces